project/sites/imtac/themes/imtac_theme/src/scss/components.scss

```diff
@@ -47,9 +49,13 @@ a.btn {
 
     p {
       @include responsive-typo-map($h3-text);
+        Display: flex;
```

Comment:
I'm sure it might still work ok, but just be careful with using capitalisation in your css, I would change these to 'display'